proposals/lip-0009.md

```diff
@@ -55,7 +55,7 @@ Note that the sum symbol “+” in the formula above stands for the string conc
 
 It is important to point out that the version and community identifier fields should be well documented and defined in the Lisk ecosystem (for example as in [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)).
 
-In order to implement the proposed change in the Lisk ecosystem (Lisk Core, Lisk Elements), the logic in the following functions needs to be updated to include the network identifier:
+In order to implement the proposed change in the Lisk ecosystem (Lisk Core, Lisk Elements), the network identifier has to be included in first position of the byte array used as the input of the transaction signing process. This will impact the following functions according to the current implementation:
```

Comment:
The getBytes function is used both for signing and generating the transaction ID. Therefore, I think it would be better to make clear that the byte array for generating transactions remains unchanged and does not include the network identifier.

Reply:
added clarification in the new commit below